ams/ams-api/src/main/java/com/netease/arctic/ams/api/client/ZookeeperService.java

```diff
@@ -0,0 +1,74 @@
+package com.netease.arctic.ams.api.client;
```

Comment:
The apache license is missing.
You can add the apache license template in the IDE setting.

Reply:
@hameizi  this class seems lack licence